index.bs

```diff
@@ -1285,6 +1293,17 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
 Its value is 64.
 
+<dfn>Default max number of event states</dfn> is a positive integer that controls the
+default [=attribution source/max number of event states=]. Its value is 3.
+
+<dfn>Max number of attribution scopes per source</dfn> is a positive integer that controls the
+maximum [=set/size=] of an [=attribution source=]'s [=attribution source/attribution scopes=].
+Its value is 20.
+
+<dfn>Max length per attribution scope</dfn> is a positive integer that controls the
+maximum [=string/length=] of an attribution scope from [=attribution source=]'s [=attribution source/attribution scopes=].
```

Comment:
Does this limit also apply to trigger’s scopes?

Reply:
Done. updated the constants to apply to both source and trigger.